installer/roles/local_docker/tasks/upgrade_postgres.yml

```diff
@@ -1,26 +1,31 @@
 ---
-- name: Check for existing Postgres data
-  stat:
-    path: "{{ postgres_data_dir }}/pgdata/PG_VERSION"
+
+- name: Register full path to postgres data dir
+  shell: "echo {{ postgres_data_dir }}"
```

Comment:
This looks a little weird. Cant this just be a regular ol variable?

Reply:
The problem is that when mounting the volume we need to provide docker the full path, it doesn't like the `~`.

Reply:
Ansible has a `| realpath` filter since 1.8.  ref: https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#managing-file-names-and-path-names

Reply:
Nice, I'll use that.